### ISSUE TITLE
Missing background width control in mobile

### ISSUE THREAD

**Issue by KiQdev** [elementor/elementor#17298]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Edit column overlay background in mobile view and set to custom size, the width control is missing
-->

## Isolating the problem

- [ x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ x] I can reproduce this bug consistently using the steps above.

## Environment

WordPress Environment |   |  
-- | -- | --
Version: | 5.8.2
![edit column missing width control](https://user-images.githubusercontent.com/30444495/147404356-d87d204e-b33c-448b-8acc-f4c8112b89da.png)

```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```<html><body>
<!--StartFragment-->

Operating System: | Linux |  
-- | -- | --
Software: | Apache |  
MySQL version: | MariaDB Server v5.7.36 |  
PHP Version: | 8.1.1 |  
PHP Max Input Vars: | 3000 |  
PHP Max Post Size: | 2G |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected

<!--EndFragment-->
</body>
</html>
</details>

**Comment by mcrowell** [comment]
+1

**Comment by gestionatuweb** [comment]
Yes, and custom background position, controls are not displayed too.

![image](https://user-images.githubusercontent.com/77508957/147468423-cd81e523-05f3-4621-a84a-eaaf3386b79d.png)

**Comment by marketingspectrumNL** [comment]
+1 
Counts for advanced -> custom positioning as well.
Example:
When desktop is set to inline and you go to the mobile view, you can select custom but no settings come up.

**Comment by lks290z** [comment]
+1

**Comment by marketingspectrumNL** [comment]
Custom positioning bug has been open for 28 days now? 

When is a fix expected?

**Comment by mcrowell** [comment]
Status? This bug is causing a delay on a client project.

**Comment by lennoxcreativeco** [comment]
+1, background positioning on mobile hasn't been working for a while and it's very frustrating!

**Comment by brianyerkes** [comment]
+1, been a long time experiencing this issue. If anyone wants a quick fix, simply give your section a class or ID, and add some css...change below to suit.
`@media (max-width: 767px) {
.bg-mobile {
background-position: 140% 20%;
background-size: 60%;
}
}`

**Comment by mcrowell** [comment]
Thanks @brianyerkes !
@rami-elementor @shilo-ey is there any sort of update on this bug?

**Comment by sinnebild** [comment]
Also having this issue.
I have noticed that creating a custom breakpoint in the latest elementor version, 3.7.2, will let you set custom x-, y-position and width for that breakpoint.
So i guess the bug arrived at the same time custom breakpoints were introduced.

**Comment by arielk** [comment]
We already fixed this original issue in v3.7, but I know we have another issue in this area, so I've merged it into #19861 for better handling
